tests/unit/test_disco_elb.py

```diff
@@ -720,7 +748,6 @@ def test_add_tags_created_tg(self):
             VpcId=TEST_VPC_ID,
             HealthCheckProtocol="HTTP",
             HealthCheckPort="80",
-            HealthCheckEnabled=True,
```

Comment:
This cannot be disabled for instances and is true by default so we don't set this anymore